api/fixtures/example.go

```diff
@@ -308,16 +312,26 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 		},
 	}
 
-	var nodePool *hyperv1.NodePool
-	if o.NodePoolReplicas > -1 {
-		nodePool = &hyperv1.NodePool{
+	if o.NodePoolReplicas <= -1 {
```

Comment:
Did you mean < 1?

Reply:
Flag description is `If >-1, create a default NodePool with this many replicas`

So I am inverting the logic here so I can exit early if the user doesn't what the NodePools created.